testStrategies/commonToAllTestStrategies.go

```diff
@@ -153,57 +133,75 @@ func (ts *TestSuite) BuildTestSuite(configurationSettings *perfTestUtils.Config)
 				continue
 			}
 
-			testDefinition, err := loadTestDefinition(bs, configurationSettings)
+			testDefinition, err := loadTestDefinition(bs)
 			if err != nil {
```

Comment:
The loadTestDefinintion function has been simplified. Found we didn't need to pass in configSettings.